### DISCUSSION TITLE
Athena id/name to img

### DISCUSSION THREAD

**Discussion by Kerchakk** [LeleDerGrasshalmi/FortniteEndpointsDocumentation#302] [Q&A] [answered]
hello, I scraped some data from query profile, now I find myself with tons of ids, is there an API endpoint that permit to convert those to an image directly or would I have to go through an unofficial API

ex id : 50c1e82f-0e69-4f7b-b6ac-e5bf8363dc16
ex name : AthenaDance:spid_225_s14_hightowertomato

**Comment by ILMCTS** [comment]
you would need gamefiles to convert the templateId's id to cosmetic data, or use an unofficial api like fortnite-api.com

**Reply by Kerchakk** [reply]
yea i figured out about this, was hopping there was some way to do it using their api, but ok i'll go for fortnite-api.com

**Reply by Kerchakk** [reply, marked as answer]
in case anyone need after myself, here is the link you could use : 
https://fortnite-api.com/images/cosmetics/br/item_id/icon.png
in this case 
https://fortnite-api.com/images/cosmetics/br/spid_225_s14_hightowertomato/icon.png
you can change 'icon.png' to 'smallicon.png' to get smaller images (faster to download)

**Reply by Kerchakk** [reply]
if someone know how to use fortnite api to get item name, rarity, and other data, i'll take it, cant manage to find if they have it.

**Reply by LeleDerGrasshalmi** [reply]
https://dash.fortnite-api.com/

**Reply by Kerchakk** [reply]
Ive actually downloaded the item db from https://fortniteapi.io/v2/items/list?lang=en to get every item id associated with their image and name so I can work on everything locally

**Reply by LeleDerGrasshalmi** [reply]
ok, could you close the discussion then as completed and mark it as the answer?

**Reply by Kerchakk** [reply]
for sure, thanks for your help
